### ISSUE TITLE
Refactor tags within GUI code to improve accessibilitiy

### ISSUE THREAD

**Issue by mobyvb** [storj/storj#4768]
We want to refactor tags within the gui code so that tags (like for buttons) are not buried in the code. This will make the GUI code readable for screen readers, solving the issue below. It is also a best practice for having cleaner code. 
______

Hello,

I use Google chrome on Debian, with Orca as I am blind.

I successfully signed up but noticed some issues:

Firstly, the satellit selection is not really accessible. Orca doesn't announce something we have to check or this is a list where we have to click. In fact, we cannot really select a satellite using keyboard only;

When I finished to fill the form, I don't know how I did but I succeeded in clicking on Subscribe. I think there was a confirmation message, but it was not rad by Orca, the screen reader. So I received plenty of mails to confirm my account;

I don't know how I clicked on the button because it is unfortunately not presented as a button for Orca. It is presented as clickable but Orca doesn't always allow to use the keyboard except if it is a button in the HTML code of the page.

I observed the same kind of things when I signed in, the sign in button is not presented as a button and the buttons to add a payment method for example is not presented as a button nor as clickable.

Would it be possible to improve these things?As I am a developer I can try to see if I can do something and provide a pull request but as I am not sure I will be able to do so I would like to know if it could be improved.

I stay of course available if you need more information.

More details and Zendesk comments at https://storjlabs.atlassian.net/browse/USR-960

Acceptance Criteria:
* Make signup and login forms in satellite UI navigable with keyboard only.

**Comment by stoweandi** [comment]
@mobyvb list for low hanging fruit items.
